docs/examples/Weather.jsx

```diff
@@ -0,0 +1,11 @@
+var example = (
+  <div className="example-weather">
+    <UIToolkit.Weather type="sunny" temperature={22} unit="c" date="2016-07-25" />
```

Comment:
I thought about forcing it, people should always use a capital C for Celsius etc.
But I though i would give people the freedom to use it how they wish. React will throw a warning to tell them they are using it wrong (check console)